package.json

```diff
@@ -50,6 +50,7 @@
 	},
 	"type": "module",
 	"dependencies": {
+		"@anthropic-ai/sdk": "^0.17.1",
```

Comment:
`@anthropic-ai/sdk` should be installed optionally, like the other inference specific packages

`npm i --save-optional @anthropic-ai/sdk`